### DISCUSSION TITLE
commit action is stuck and frozen for 30+ minutes

### DISCUSSION THREAD

**Discussion by scarf005** [stefanzweifel/git-auto-commit-action#345] [Help] [answered]
https://github.com/cataclysmbnteam/Cataclysm-BN/actions/runs/11154642824/job/31004269727

![image](https://github.com/user-attachments/assets/67ba743c-dbe8-4935-bcb2-4fbc5aac8f7a)

workflow file used: https://github.com/cataclysmbnteam/Cataclysm-BN/blob/7a201db13964fda87019b9686e44327c8df71337/.github/workflows/pull-translations.yml

not sure why this is happening, did i miss a step? help would be appreciated.

**Comment by stefanzweifel** [comment]
@scarf005 the workflow summary shows errors and warnings: https://github.com/cataclysmbnteam/Cataclysm-BN/actions/runs/11154642824

Looks like the runner is running out of disk-space and something consumed all CPU cycles and memory.
Maybe this was a one-off problem (happens from time to time in my workflows as well, that GitHub Actions has a problem), then we can't do anything.

Can you maybe test, that the code running in your workflow works locally? git-auto-commit is just a bash script that runs different git-commands. Maybe you run into an edge-case in git itself.

**Reply by scarf005** [reply]
https://github.com/cataclysmbnteam/Cataclysm-BN/actions/runs/11154642824/job/31010195802

the issue keeps happening after re-running.

https://github.com/cataclysmbnteam/Cataclysm-BN/blob/7a201db13964fda87019b9686e44327c8df71337/.github/workflows/pull-translations.yml#L23-L27

maybe this line is related?

**Reply by scarf005** [reply, marked as answer]
https://github.com/scarf005/Cataclysm-BN/commit/5aa8b4ee185eeece4c6730287bd22973e8747694

yeah, commit action seems to choke on sparse checkout. removing the line made it work again.

**Reply by stefanzweifel** [reply]
Great to here that you found a solution. 🎉
